Translations/de.lproj/Localizable.strings

```diff
@@ -838,6 +838,15 @@
 /* The placeholder for URL Field when saving a custom search engine */
 "Settings.AddCustomEngine.URLPlaceholder" = "URL (Anfrage durch %s ersetzen)";
 
+/* The button text in Search Settings that opens the list of supported search languages. */
+"Settings.SearchResultForLanguage" = "";
```

Comment:
@when we should add the localizations. @chrmod  maybe we ask someone now?

Reply:
Done